src/test/java/org/collectionspace/qa/cucumber/supplementary_to_primary/SupplementaryPrimaryIT.java

```diff
@@ -1,4 +1,4 @@
-package org.collectionspace.qa.cucumber.supplementary_to_primary;
+package org.collectionspace.qa.cucumber.supplementary_to_primary_tab;
```

Comment:
This package name needed to be changed slightly to match the folder hierarchy in the project; the relevant feature files are (as of this writing) in a 'supplementary_to_primary_tab' folder.